compressor/storage.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import unicode_literals
 import errno
 import gzip
+import brotli
```

Comment:
this imports unconditionally, but it is not listed in the requirements in setup.py. Since most other dependencies we have are optional, i suggest to do the same here, an example is here: https://github.com/django-compressor/django-compressor/blob/develop/compressor/parser/beautifulsoup.py#L13